### PR TITLE
Some cleanups

### PR DESCRIPTION
**Pull request by wo0dyn**
## Checklist

* [x] balise → tag
* [x] SeparatorWidget → `<hr>`?
* [x] Fix English « invalide »
* [x] SeparatorFieldSerializer → fix comments
* [x]  res + append → yield
* [x]  Format strings
* [x]  Fix imports (use isort?)

## Note for the review

_The final diff from “Files changed” is pretty large, so please review commit by commit._

## Note about isort

I added a test in out `circle.yml` to check imports. I display a full output to get all information needed to debug. It prints the configuration used as well.

```console
$ make check-python-imports

✓ isort installed
[isort]

known_django = django
known_first_party = formidable
sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER

combine_as_imports = true
multi_line_output = 5 ; Hanging Grid Grouped
not_skip = __init__.py
skip = migrations, settings
line_length = 80
isort --check-only --recursive --verbose formidable

/#######################################################################\

     `sMMy`
     .yyyy-                                                      `
    ##soos##                                                    ./o.
          `     ``..-..`         ``...`.``         `   ```` ``-ssso```
     .s:-y-   .+osssssso/.     ./ossss+:so+:`    :+o-`/osso:+sssssssso/
     .s::y-   osss+.``.``     -ssss+-.`-ossso`   ssssso/::..::+ssss:::.
     .s::y-   /ssss+//:-.`   `ssss+     `ssss+   sssso`       :ssss`
     .s::y-   `-/+oossssso/  `ssss/      sssso   ssss/        :ssss`
     .y-/y-       ````:ssss`  ossso.    :ssss:   ssss/        :ssss.
     `/so:`    `-//::/osss+   `+ssss+-/ossso:    /sso-        `osssso/.
       \/      `-/oooo++/-      .:/++:/++/-`      ..           `://++/.


         isort your Python imports for you so you don't have to

                            VERSION 4.2.5

\########################################################################/

SUCCESS: /home/ubuntu/django-formidable/formidable/__init__.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/accesses.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/constants.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/models.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/register.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/urls.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/validators.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/views.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/__init__.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/boundfield.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/field_builder.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/fields.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/widgets.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/validations/__init__.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/forms/validations/presets.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/__init__.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/access.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/child_proxy.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/common.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/defaults.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/fields.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/forms.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/items.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/list.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/presets.py Everything Looks Good!
SUCCESS: /home/ubuntu/django-formidable/formidable/serializers/validation.py Everything Looks Good!
WARNING: migrations was skipped as it's listed in 'skip' setting or matches a glob in 'skip_glob' setting
Skipped 1 files
```

### CODE REVIEW

File: formidable/forms/field_builder.py [hunk 1]
```diff
@@ -89,11 +89,9 @@ def get_help_text(self):
         return self.field.help_text
 
     def get_validators(self):
```
Comment (aRkadeFR) [inline formidable/forms/field_builder.py]: I would not expect to have an iterator with the name ``get_validators``

Comment (wo0dyn) [inline formidable/forms/field_builder.py]: What name should we use instead of `get_validators()`?

Comment (moumoutte) [inline formidable/forms/field_builder.py]: get_validators with iterators doesn't bother me. I have more difficulties with build_rules method

Comment (aRkadeFR) [inline formidable/forms/field_builder.py]: I agree on build_rules too.
If I want all the validators as the name ``get_validators`` suggests, I need to cast it to list() for instance in order to have them all in python2. I would change it for gen_validators and gen_rules (for generator)

Comment (wo0dyn) [inline formidable/forms/field_builder.py]: I can revert the commit if you guys are not comfortable with.

Comment (moumoutte) [inline formidable/forms/field_builder.py]: You can keep going with this idea.

Like @aRkadeFR said, you can wrap  ( I understand correctly) gen_validators with list(gen_validators()) on get_validators. Same thing for build_rules.

**Comment by moumoutte** [comment]
Many Thanks !

**Pull request merged**